webpack.config.js

```diff
@@ -63,6 +63,18 @@ const config = {
 };
 
 module.exports = (env) => {
+
+  if (env.onPrem) {
+    console.log("building for onprem");
+    config.entry.extension = "./src/extensionOnPrem.ts";
```

Comment:
this is an uncommon practice to do it , you can just modify the https://github.com/codota/tabnine-vscode/blob/dc35e8a59c7ecd0e074c893fc0a58f5bfb3e1c20/webpack.config.js#L14
and do something like ``"./src/extension${env.onPrem ? "OnPrem" : ""}.ts"``

Reply:
there is no env object there